testsuite/tests/contrib/lkt_semantic/array_type_concrete/test.lkt

```diff
@@ -0,0 +1,15 @@
+val i: Int = 1
+# TODO: should infer type from i
+val a = [i, 2, 3]
+
+val s: String = "str1"
+# TODO: should infer type from s
+val a = [str, "str2"]
```

Comment:
Typo: should be `s` (not `str`)

Reply:
Correct, and the test.out below tells me that also, sorry to missed that.